output/plugins/forms/xml/forms.cppcode

```diff
@@ -407,7 +407,26 @@ Written by
 			#ifequal $aui_managed "1"
 			@{ @#include &lt;wx/aui/aui.h&gt; @}
 		</template>
-		<template name="settings">
+        <template name="code_init">
```

Comment:
This name doesn't match the present naming style, the name components are in reversed order

Reply:
<snip>
> Is there any reason why you are still moving the initialization code off the parameterized constructor into an own method? The delayed execution of the wxFB generated code can be achieved by using two-step creation. I think supporting this standard approach of wxWidgets is of greater value than just refactoring the one-step creation approach.
<snip>
 
I want to have the class created normally with all the parameters passed, but I don't want any children created before setting the font. If I don't do this explicit calls will have to be made to functions to do what can be done by just passing parameters at instantiation. To me it is cleaner. It also allows other calls to be made if needed before creating extra child objects.

Reply:
But this is what two-step creation enables you to do. Keep in mind you don't use the generated class directly, instead you derive a class from it and use that one. With two-step creation you can define a parameterized constructor in your derived class but still default create the (generated) base class. Before you call the Create() method you can call other methods first, e.g. change the font. Granted, if you don't call Create() in that constructor, you need to save some arguments to use them later.

I just merged the two-step creation feature because it is a nice addition to wxFB. There you can see that no additional code templates are required (minus the one that gets executed after event handler connection, but this might be useful for future enhancements too). Also this is kind of the state i would a PR expect to reach to be able to merge it.

If you still need that Init() approach i would suggest to make it an alternative to two-step creation because i don't think it's easy to support both together nor does this make sense. For that purpose i would suggest to replace the `two_step_creation` boolean property with an option property, maybe `delayed_construction`, that has the values `<empty>` for one-step creation only, `two-step` for two-step creation and `init` for your proposed Init() method.

Reply:
This has the problem that for every change, i.e. new methods that need to be called (style is not the only one that may be needed, and android proved that) that may be needed between instantiation of the class and creating all the children you will have to make extra changes to the Create class. My version achieves the same but allows the developer to change the parts they require before building all the children and therefore if there are other objects/settings/etc. that need changing they can be. The resultant code is almost the same but with the Create you have lock in and inbuilt maintenance.

Reply:
This change also does not change the default style before creating child objects. The whole point of this change was to allow changes to be made to the default settings prior to them being instantiated. In Android if you do not set the default style to what is required before creating children the only way to change the style is to update each child element individually. So the change I put is has the desired effect, so I will update my pull request with your code so that it can be pulled in.

Reply:
But this is exactly what two-step creation is made for. I don't know how i can explain the situation better, so im trying to show you two simple examples instead. Just to make sure, the usual process to use the wxFB generated class is not to use the class directly but inherit from it. With enabled two-step creation you have basically two choices how to modify the parent class before the generated code adds the components.

Example 1: In-Class modification inside derived class
```c++
// Generated by wxFB
class MyGeneratedDialog;

class MyDialog : public MyGeneratedDialog {
public:
   MyDialog(wxWindow* parent, wxWindowID id = wxID_ANY, const wxString& title = wxEmptyString, const wxPoint& pos = wxDefaultPosition, const wxSize& size = wxDefaultSize, long style = wxDEFAULT_DIALOG_STYLE) {
      SetFont(*wxITALIC_FONT);
      SetExtraStyle(wxWS_EX_PROCESS_IDLE);
      SetForegroundColour(*wxRED);

      Create(parent, id, title, pos, size, style);
   }
};

int main() {
   auto* dlg = new MyDialog(nullptr);

   return 0;
}
```

Example 2: External modification
```c++
// Generated by wxFB
class MyGeneratedDialog;

class MyDialog : public MyGeneratedDialog {};

int main() {
   auto* dlg = new MyDialog();
   dlg->SetFont(*wxITALIC_FONT);
   dlg->SetExtraStyle(wxWS_EX_PROCESS_IDLE);
   dlg->SetForegroundColour(*wxRED);

   dlg->Create(nullptr);

   return 0;
}
```

Why can't you use any of these approaches?